.github/workflows/ci-tests-n-coverage.yml

```diff
@@ -0,0 +1,126 @@
+name: Testing and Code Coverage
+
+on:
```

Comment:
This a friendly reminder that yesterday we agreed to tun this job nightly on main.

Reply:
Sorry, had forgotten to add this. Thanks.